src/server/handlers.ts

```diff
@@ -282,7 +282,7 @@ export const SendMessageHandler = (socket: Socket) => {
 
       const messageDetails = {
         ...sentMessage,
-        room: chatId,
+        chatId,
```

Comment:
Why not `room`? This introduces inconsistency hence a bug on the frontend, because type `MessageType` expects `room` which is what we expect to be returned when the server ackowledge the message 

@MJinH would you mind explaining?

MessageType is defined here:
https://github.com/Dun-sin/Whisper/blob/b73c6db38c94ebe052d8aaf512379a4a1401f4de/src/types/types.ts#L40

Usage is here:
https://github.com/Dun-sin/Whisper/blob/b73c6db38c94ebe052d8aaf512379a4a1401f4de/src/lib/chatSocket.ts#L21

Reply:
i asked him to change it because frontend uses chatId

Reply:
let's just merge this somewhere else(different branch), then i will change everything to say either room or chatId

Reply:
I agree.